packages/material-ui/src/FormLabel/FormLabel.test.js

```diff
@@ -4,6 +4,8 @@ import { expect } from 'chai';
 import { createMount, describeConformanceV5, act, createClientRender } from 'test/utils';
 import FormLabel, { formLabelClasses as classes } from '@material-ui/core/FormLabel';
 import FormControl, { useFormControl } from '@material-ui/core/FormControl';
+import { hexToRgb } from '@material-ui/core/styles';
+import defaultTheme from '../styles/defaultTheme';
```

Comment:
use `defaultTheme` here to remove hard coded color in the test below.

I wonder should `defaultTheme` be exported as `defaultLightTheme` from `@material-ui/core/styles` also?

In my side project I use `defaultTheme` a lot and it is quite annoying to call `createTheme()` everytime.